app/models/concerns/hyku_addons/account_behavior.rb

```diff
@@ -140,5 +164,21 @@ def set_smtp_settings
           HykuAddons::PerTenantSmtpInterceptor.available_smtp_fields.each_with_object("").to_h
         )
       end
+
+      def set_shared_search_default
+        return if settings['shared_search'].present?
+        self.shared_search = false
```

Comment:
I think you can safely change both lines for:
```suggestion
        self.shared_search ||= false
```